src/bert/utils.py

```diff
@@ -0,0 +1,106 @@
+import os
+import random
+import copy
+import logging as log
+
+from typing import List, Dict
+
+import numpy as np
```

Comment:
[nit] some of these imports aren't used -- e.g. `os`, `copy`, `List`, `numpy`, or `random`. We probably have many more cases of this in the codebase. Perhaps the one to be most wary of is where we call/import `random`

Reply:
Yeah, some of this was copypasta from the openai model wrapper, which requires a lot more preprocessing. Fixed.